src/browser/components/icons/Icons.tsx

```diff
@@ -421,12 +421,20 @@ export const FireExtinguisherIcon = ({
   <IconContainer className="fa fa-fire-extinguisher" title={title} />
 )
 
-export const NewFolderIcon = () => <IconContainer icon={newFolder} width={12} />
-export const NavIcon = () => <IconContainer icon={navIcon} width={12} />
-export const AddIcon = () => <IconContainer icon={addCircle} width={12} />
-export const FolderIcon = () => <IconContainer icon={folder} width={12} />
-export const SavedScriptsPlay = () => (
-  <IconContainer icon={run_icon} width={12} />
+export const NewFolderIcon = (): JSX.Element => (
```

Comment:
There's a lot of duplication in creating all these IconContainer wrappers.

What do you think of the idea of having helpers to clean that up?


Something like:
```
const createIconContainer = props => <IconContainer ...props />
const createSmallIconContainer = props => createIconContainer({...props, width: SMALL_SIZE})
const createLargeIconContainer = props => createIconContainer({...props, width: LARGE_SIZE})
```

If nothing else, this might remove the need for all the TypeScript types?

Reply:
Hmm, the typescript types are there since we have a rule on fully specified In/out values for things that are exported. I'm open to letting return values be inferred even on exported functions, but that's another discussion

Reply:
I would want to do helpers like these in the future, but I feel it's a premature optimization to do so now as we have such a large spread of ways icons are done in the code base (via several fonts, semantic UI, as svg files and a few more). So I think another layer of indirection isn't helpful until we've standardized our iconhandling

Reply:
Yeah, makes sense to me! Thanks for clarifying your thoughts :-)